draft-ietf-lamps-dilithium-certificates.xml

```diff
@@ -119,6 +119,12 @@
 	      <xref target="RFC3279" format="default"></xref> did for classic cryptography and
 	      <xref target="RFC5480" format="default"></xref> did for elliptic curve cryptography.
 	      The private key format is also specified.</t>
+      <t><xref target="FIPS204" format="default"></xref> defines two versions of ML-DSA, the pure and the 
+	      pre-digest version. Only the former is specified for use in this document. 
```

Comment:
I propose we remove "The size [...]  ML-DSA performance.", and add

> Pure ML-DSA (in contrast to SLH-DSA) allows for streaming the message that has to be signed. The only advantage of the pre-digest version, is that it allows a different hash to be used for hashing the message.

Reply:
The pre-digest people do not argue that you cannot stream the message. Their point is that maybe I don't want to stream it to the HSM over its slow bus. I may want to just send the digest of the message and sign that especially if the message will be signed a lot of times.